BraintreeCore/src/test/java/com/braintreepayments/api/ConfigurationUnitTest.kt

```diff
@@ -0,0 +1,479 @@
+package com.braintreepayments.api
+
+import android.text.TextUtils
+import org.json.JSONException
+import org.junit.Assert.*
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.util.*
+
+@RunWith(RobolectricTestRunner::class)
+class ConfigurationUnitTest {
```

Comment:
Maybe a silly question but more for my own knowledge. Does Kotlin require us to convert the tests alongside the files?

Reply:
No it doesnt. Kotlin and java files can live together, you can have a mix of them in whatever form you like.
We are changing the tests becuase well, Kotlin is the better language hehe but also, because this can allows us to use [Mockk](https://mockk.io/), which is a very powerfull library for mocking kotlin classes

Reply:
Moving to Mockk improves code quality as well. Powermock and Robolectric don't work well together, and we've wanted to get rid of Mockito for a really long time.